test/web-platform-tests/to-upstream/html/webappapis/timers/settimeout-setinterval-handles.html

```diff
@@ -9,6 +9,21 @@
 <script>
 "use strict";
 
+test(() => {
+  const a = setTimeout(noop);
+  const b = setInterval(noop);
+
```

Comment:
It feels nicer to clearTimeout/clearInterval to clean up after this test, but I guess the test runner should be able to handle it even if you don't...